Templates/AppSource App/.github/workflows/PullRequestHandler.yaml

```diff
@@ -38,6 +38,7 @@ jobs:
       telemetryScopeJson: ${{ steps.init.outputs.telemetryScopeJson }}
       githubRunner: ${{ steps.ReadSettings.outputs.GitHubRunnerJson }}
       githubRunnerShell: ${{ steps.ReadSettings.outputs.GitHubRunnerShell }}
+      checkoutRef: ${{ github.event.pull_request.merge_commit_sha }}
       projects: ${{ steps.determineProjectsToBuild.outputs.ProjectsJson }}
```

Comment:
I think you placed this in the wrong location (under outputs) - should be a parameter for _BuildALGo...

Reply:
Yes, yes... figured it out the long way 😄